README.md

```diff
@@ -173,8 +171,6 @@ jobs:
       - name: Load AWS credentials
         uses: 1password/load-secrets-action@v1
         with:
-          # Export loaded secrets as environment variables
-          export-env: true
```

Comment:
Could be good to still keep these in the README for documentation purposes.

Reply:
Sure, will keep them in the docs then.